services/test-validators.js

```diff
@@ -155,6 +155,15 @@ const isCustomCompactTestTotals = makeCompactTestTotalsValidator({
   skipped: '🤷',
 })
 
+const isOrdinalNumber = Joi.string().regex(/^[1-9][0-9]*(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ)$/)
```

Comment:
Since we're in the subject of regexes, I modified also `isOrdinalNumber`, since for example the value `1ˢᵗ` did not pass the tests with the previous regex (since it required a digit from 1 to 9, followed by one or more digits from 0 to 9 :  `[1-9][0-9]+`).

I changed this:
`const isOrdinalNumber = Joi.string().regex(/^[1-9][0-9]+(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ)$/)`
to this:
`const isOrdinalNumber = Joi.string().regex(/^[1-9][0-9]*(ᵗʰ|ˢᵗ|ⁿᵈ|ʳᵈ)$/)`

The only thing that I changed was a `+` to an `*`.